docs/Concepts/OOP/OOP.md

```diff
@@ -1,33 +1,252 @@
 ---
 layout: default
 title: OOP
-description: "Warning: The following page is currently under construction, find more about the details in future patches, or if you choose to add in the article see info on the bottom of the page."
+description: An article dicussing Object Oriented Programming.
 nav_order: 1
-parent: Concepts
+parent: OOP
```

Comment:
It can not be a parent of itself, for referencing parents or grandparents is not related to the folder name, but rather the File name that it sits in.

OOP (Folder name) , OOP.md (File Name)  Meaning the Parent of this file is referencing itself as the folder holder, though the parent where that folder sits in is within Concepts. 

If you were to make a file  " NewFileOOP " within the OOP (Folder) then the parent would be OOP and grandparent would be " Concepts ".

Reply:
Has been Resolved for both issues, though will leave for reference in the future.